tdns/miek_utils.go

```
@@ -10,161 +10,155 @@ import (
-                                
+
 func dddToByte[T ~[]byte | ~string](s T) byte {
-        _ = s[2] // bounds check hint to compiler; see golang.org/issue/14808
-        return byte((s[0]-'0')*100 + (s[1]-'0')*10 + (s[2] - '0'))
+	_ = s[2] // bounds check hint to compiler; see golang.org/issue/14808
+	return byte((s[0]-'0')*100 + (s[1]-'0')*10 + (s[2] - '0'))
 }
-                
+
 // Helper function for packing and unpacking
 func intToBytes(i *big.Int, length int) []byte {
```

golangci-lint (1.62.2)

func `intToBytes` is unused

(unused)